src/Areas/Canvases/Models/WorkbookInfo.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace AzureMcp.Areas.Canvases.Models;
+
+public sealed record WorkbookInfo(
+    string WorkbookId,
+    // Display name can be null if not set.
+    string? WorkbookDisplayName,
```

Comment:
Does this need to be prefixed with Workbook or does DisplayName work?

Also probably don't need the comment above.